### DISCUSSION TITLE
07.00.06 (Release Candidate 1)

### DISCUSSION THREAD

**Discussion by WillStrohl** [DNNCommunity/Dnn.CommunityForums#261] [Announcements]
# Community Forums 07.00.06 (RC1)  

**IMPORTANT!!!**  This is a release candidate (RC).  Please do not use this installation/upgrade package for any production websites of any kind.  This release is not supported and is only provided for test purposes, to allow others to test their websites first.  

## Features/Bug Fixes/Enhancements  

* NEW: Replaced the Deprecated DNN Platform Method `GetModuleSettings()` with the Replacement `.ModuleSettings` Property ([Issue 252](https://github.com/DNNCommunity/Dnn.CommunityForums/issues/252), thanks @johnhenley!)  
* BUG: Hidden forum groups not showing if some forums in the hidden group are visible ([Issue 243](https://github.com/DNNCommunity/Dnn.CommunityForums/issues/243), thanks @johnhenley!)  
* BUG: Fixed the Order of "What's New" (Companion) Module w/ Topics-Only Option ([Issue 205](https://github.com/DNNCommunity/Dnn.CommunityForums/issues/205), thanks @johnhenley!)  
* TASK: Replaced Hardcoded Paths/Files to Constants ([Issue 224](https://github.com/DNNCommunity/Dnn.CommunityForums/issues/224), thanks @johnhenley!)   
<hr />

## Download  
[Community Forums 07.00.06 RC 1](https://github.com/DNNCommunity/Dnn.CommunityForums/releases/tag/07.00.06-RC1)  

<hr /><em>This discussion was created from the release <a href='https://github.com/DNNCommunity/Dnn.CommunityForums/releases/tag/07.00.06-RC1'>07.00.06 (Release Candidate 1)</a>.</em>

**Comment by johnhenley** [comment]
@WillStrohl I found a few bugs, so I put in PR #276
